helm/prometheus-agent-app/values.yaml

```diff
@@ -0,0 +1,15 @@
+remoteWrite:
+ - url: http://thanos-receive-ingestor-default.thanos.svc:19291/api/v1/receive
+
+labelSelector:
+  app: prometheus-agent
+
+image: quay.io/giantswarm/prometheus
```

Comment:
This does not allow any override of the registry :D 
Something like:
image:
  registry: docker.io
  name: giantswarm/prometheus
  # -- Overrides the image tag whose default is the chart's appVersion
  tag: ""
  sha: ""